<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simple Cities</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadRoads"/>
    <addaction name="actionSaveRoads"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionSaveImage"/>
    <addaction name="actionSaveImageHD"/>
    <addaction name="separator"/>
    <addaction name="actionLoadCamera"/>
    <addaction name="actionSaveCamera"/>
    <addaction name="actionResetCamera"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPM">
    <property name="title">
     <string>PM</string>
    </property>
    <addaction name="actionGenerateBlocks"/>
    <addaction name="actionGenerateParcels"/>
    <addaction name="actionGenerateBuildings"/>
    <addaction name="actionGenerateVegetation"/>
    <addaction name="actionGenerateAll"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionView2D"/>
    <addaction name="actionView3D"/>
    <addaction name="separator"/>
    <addaction name="actionShowWater"/>
    <addaction name="actionSeaLevel"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPM"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewTerrain">
   <property name="text">
    <string>New Terrain</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionOpenTerrain">
   <property name="text">
    <string>Open Terrain</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionControlWidget">
   <property name="text">
    <string>Control Widget</string>
   </property>
  </action>
  <action name="actionSaveTerrain">
   <property name="text">
    <string>Save Terrain</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionSaveRoads">
   <property name="icon">
    <iconset>
     <normaloff>Resources/save.png</normaloff>Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Roads</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPropertyWidget">
   <property name="text">
    <string>Property Widget</string>
   </property>
  </action>
  <action name="actionLoadRoads">
   <property name="icon">
    <iconset>
     <normaloff>Resources/open.png</normaloff>Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Load Roads</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionLoadCamera">
   <property name="text">
    <string>Load Camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionSaveCamera">
   <property name="text">
    <string>Save Camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionSaveImageHD">
   <property name="text">
    <string>Save Image HD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionResetCamera">
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionGenerateBlocks">
   <property name="text">
    <string>Generate Blocks</string>
   </property>
  </action>
  <action name="actionGenerateParcels">
   <property name="text">
    <string>Generate Parcels</string>
   </property>
  </action>
  <action name="actionGenerateBuildings">
   <property name="text">
    <string>Generate Buildings</string>
   </property>
  </action>
  <action name="actionGenerateVegetation">
   <property name="text">
    <string>Generate Vegetation</string>
   </property>
  </action>
  <action name="actionGenerateAll">
   <property name="text">
    <string>Generate All</string>
   </property>
  </action>
  <action name="actionLoadParcels">
   <property name="text">
    <string>Load Parcels</string>
   </property>
  </action>
  <action name="actionLoadBuildings">
   <property name="text">
    <string>Load Buildings</string>
   </property>
  </action>
  <action name="actionGenerateRoads">
   <property name="text">
    <string>Generate Roads</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLoadZone">
   <property name="text">
    <string>Load Zone</string>
   </property>
  </action>
  <action name="actionSaveParcels">
   <property name="text">
    <string>Save Parcels</string>
   </property>
  </action>
  <action name="actionSaveBuildings">
   <property name="text">
    <string>Save Buildings</string>
   </property>
  </action>
  <action name="actionGenerateScenarios">
   <property name="text">
    <string>Generate Scenarios</string>
   </property>
  </action>
  <action name="actionView2D">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2D</string>
   </property>
  </action>
  <action name="actionView3D">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
  </action>
  <action name="actionSeaLevel">
   <property name="text">
    <string>Sea Level</string>
   </property>
  </action>
  <action name="actionShowWater">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Water</string>
   </property>
  </action>
  <action name="actionLoadOSMRoads">
   <property name="text">
    <string>Load OSM Roads</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
